### ISSUE TITLE
timecontrol pgn header when time control is fixed time per move

### ISSUE THREAD

**Issue by gahtan-syarif** [Disservin/fast-chess#300]
say when st=1.5, in cutechess the pgn header would be:
```
[TimeControl "1.5/move"]
```
currently in fastchess when st is set it would just display
```
[TimeControl "0"]
```

**Comment by Disservin** [comment]
mh what does it do when both players have different time controls?

**Comment by Disservin** [comment]
"fixed with https://github.com/Disservin/fast-chess/commit/217a37b7001011b2e83557b05f45eca3c32d20e7
but still what happens with two different time controls?

**Comment by gahtan-syarif** [comment]
> "fixed with [217a37b](https://github.com/Disservin/fast-chess/commit/217a37b7001011b2e83557b05f45eca3c32d20e7) but still what happens with two different time controls?

the way it should be written is like this:
```
[TimeControl "5+0.05; 10+0.1"]
```
the left side being the white and the right side being the black time control.
in fastchess right now it only displays the time control for the white player i think @Disservin
